### PR TITLE
Jv spring boot project hw 2

### PR DESCRIPTION
**Pull request by ihorvasylenko23**

### CODE REVIEW

File: src/main/java/mate/academy/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package mate.academy.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+
```
Comment (padu12) [inline src/main/java/mate/academy/dto/CreateBookRequestDto.java:8]:
```suggestion

```

File: checkstyle.xml [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        </module>
+        <module name="EmptyCatchBlock">
+            <property name="exceptionVariableName" value="expected"/>
+        </module>
+        <module name="CommentsIndentation"/>
+        <module name="DeclarationOrder"/>
+        <module name="SingleSpaceSeparator">
+            <property name="validateComments" value="true"/>
+        </module>
+    </module>
```
Comment (padu12) [inline checkstyle.xml]: files should end with empty line

```suggestion
    </module>

```

**Review by padu12 — APPROVED** [review]
LGTM!

File: checkstyle.xml [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+</module>
```
Comment (vovasalyha) [inline checkstyle.xml]: empty line

**Review by vovasalyha — APPROVED** [review]
Good. But why did you put all changes (including initial project setup) in PR related to spring web?

**Pull request merged**